src/components/navigation/NavigationMenu.vue

```diff
@@ -80,65 +77,53 @@ import {
  } from '../../utils/LaunchUtils';
 
 @Component
-    export default class NavigationMenu extends Vue {
+export default class NavigationMenu extends Vue {
+    private activeGame!: Game;
+    private contextProfile: Profile | null = null;
+    private LaunchMode = LaunchMode;
 
-        private activeGame!: Game;
-        private contextProfile: Profile | null = null;
-        private LaunchMode = LaunchMode;
-
-        @Prop({default: ""})
-        private view!: string;
-
-        get settings() {
-            return ManagerSettings.getSingleton(this.activeGame);
-        }
-
-        get thunderstoreModList() {
-            return this.$store.state.thunderstoreModList || [];
-        }
+    get thunderstoreModList() {
+        return this.$store.state.thunderstoreModList || [];
+    }
 
-        get localModList(): ManifestV2[] {
-            return this.$store.state.localModList || [];
-        }
+    get localModList(): ManifestV2[] {
+        return this.$store.state.localModList || [];
+    }
 
-        get appName(): string {
-            return ManagerInformation.APP_NAME;
-        }
+    async launch(mode: LaunchMode) {
+        try {
+            await setGameDirIfUnset(this.activeGame);
+            await throwIfNoGameDir(this.activeGame);
 
-        get canShowConfigEditor() {
-            return this.activeGame.packageLoader === PackageLoader.BEPINEX;
-        }
+            if (mode === LaunchMode.MODDED) {
+                await linkProfileFiles(this.activeGame, this.contextProfile!);
+            }
 
-        emitClick(element: any) {
-            this.$emit("clicked-" + element.getAttribute("data-ref"));
+            this.$store.commit("openGameRunningModal");
+            await launch(this.activeGame, this.contextProfile!, mode);
+        } catch (error) {
+            if (error instanceof R2Error) {
+                this.$store.commit("closeGameRunningModal");
+                this.$emit("error", error);
+            } else {
+                throw error;
+            }
         }
 
-        async launch(mode: LaunchMode) {
-            try {
-                await setGameDirIfUnset(this.activeGame);
-                await throwIfNoGameDir(this.activeGame);
+    }
 
-                if (mode === LaunchMode.MODDED) {
-                    await linkProfileFiles(this.activeGame, this.contextProfile!);
-                }
+    created() {
+        this.activeGame = GameManager.activeGame;
+        this.contextProfile = Profile.getActiveProfile();
+    }
+}
 
-                this.$store.commit("openGameRunningModal");
-                await launch(this.activeGame, this.contextProfile!, mode);
-            } catch (error) {
-                if (error instanceof R2Error) {
-                    this.$store.commit("closeGameRunningModal");
-                    this.$emit("error", error);
-                } else {
-                    throw error;
-                }
-            }
+</script>
 
-        }
+<style lang="scss">
```

Comment:
Not 100% sure here but you may be able to change this to `<style lang="scss" scoped>` to keep the CSS strictly related to this component only

Reply:
Nice, I wasn't aware that such a thing existed. Although I had to also add `::v-deep` [here](https://github.com/ebkr/r2modmanPlus/pull/889/files#diff-534c3ccf44736098d36e3858bdfb2bf282b0ece6e9f094aa8602c700a13cd64bR130) as per the [documentation](https://vue-loader.vuejs.org/guide/scoped-css.html#deep-selectors) for the selector to work after that.